### PR TITLE
use activity feed in contributions drawers

### LINKED ISSUE #2066: Change Give page panel contributions to activity feed
Change the current view of Contributions on the Give page panel to be a list of contributions via the Activity Feed view.

Show all contributions for the given user with this criteria:
- all contributions since the last ended epoch
- only contribution activity feed types
- show emoji bar too (by default this will show on Contribution component)
```
<ActivityList
  queryKey={['give-contributions', profile.id]}
  where={{
    _and: [
      { action: { _eq: 'contributions_insert'} },
      { actor_profile_id: { _eq: profile.id } },
      { circle_id: { _eq: circle.id } },
      { created_at: { _gt: last_epoch.end_date } },
    ],
  }}
/>
```

This will replace the code in GiveDrawer (for other users): https://github.com/coordinape/coordinape/blob/main/src/pages/GivePage/GiveDrawer.tsx#L77

and also the EpochStatementDrawer (for yourself)

https://github.com/coordinape/coordinape/blob/main/src/pages/GivePage/EpochStatementDrawer.tsx

### PR DESCRIPTION
**Pull request by sshmm**
## Motivation and Context

resolves #2066 

## Description

1- Used ActivityList component insider EpochStatementDrawer and GiveDrawer instead of Contribution Component
2- Added drawer prob to ActivityList in order to style for the drawers

## Test and Deployment Plan

1- in an circle with an active epoch add few contributions (through contribtutions page and by deework)
2- Observe the epochStatementDrawer and the GiveDrawer by another member

## Screenshots (if appropriate):
image

### CODE REVIEW

**Review by levity — APPROVED** [review]
Code looks good but I haven't tested the UX yet

**Comment by levity** [comment]
@wingfeatherwave do you want to give this a run-thru for visual/UX QA?

**Comment by crabsinger** [comment]
@sshmm needs a rebase (and has aconflict)

**Comment by wingfeatherwave** [comment]
I pushed up a minor UI fix.  But there seems to be an issue with the previously created contributions.  I see it with the `yarn db-seed-fresh` data.  Perhaps the seed data just needs to change?  There could be a bug with viewing older contributions, though.  See image:
 
Screenshot 2023-04-13 at 11 03 46 AM

File: api/hasura/cron/historicalActivity.ts [hunk 1]
```diff
@@ -17,8 +17,8 @@ import { insertEpochEndActivity, insertEpochStartActivity } from './epochs';
 
 Settings.defaultZone = 'utc';
 
-const START_DATE = `2022-10-18`;
```
Comment (crabsinger) [inline api/hasura/cron/historicalActivity.ts:20]: @wingfeatherwave i think we want to leave these changes out.

File: src/features/activities/ActivityList.tsx [hunk 2]
```diff
@@ -41,6 +43,8 @@ export const ActivityList = ({
       observer.observe(element);
       return () => observer.unobserve(element);
     }
+    // eslint-disable-next-line no-console
```
Comment (crabsinger) [inline src/features/activities/ActivityList.tsx:46]: @wingfeatherwave and this

File: api/hasura/cron/historicalActivity.ts (hunk 1 above)
Comment (crabsinger) [reply api/hasura/cron/historicalActivity.ts:20]: nevermind, reading this bad, you're removing this. durr

File: src/features/activities/ActivityList.tsx (hunk 2 above)
Comment (crabsinger) [reply src/features/activities/ActivityList.tsx:46]: nvm

Comment (crabsinger) [reply src/features/activities/ActivityList.tsx:46]: wait, we do want to remove this one

**Review by wingfeatherwave — APPROVED** [review]

**Pull request merged**